web3/shh.py

```diff
@@ -1,79 +1,179 @@
-from web3._utils.filters import (
-    ShhFilter,
-)
-from web3.module import (
-    Module,
+from web3.method import (
```

Comment:
It might be good to move this whole file to `utils` now, since nothing in here works without being assigned to a module. People might think that `web3.ssh.*` is part of the api.

Reply:
If we're going to do it that way, the `mungers` property on the `Method` class needs to be converted to a tuple as well as auditing it for any other mutable properties to ensure that modifications to a `Method` object don't have unintended side effects.

Reply:
> as well as auditing it for any other mutable properties to ensure that modifications to a `Method` object don't have unintended side effects.

Can you explain this a little further - what are some potential side effects? what are the other mutable properties we should check?